site2/website/release-notes.md

```diff
@@ -1,6 +1,191 @@
+* [Broker] Expose bookkeeper expose explicit lac configuration in broker.conf [#5822](https://github.com/apache/pulsar/pull/5822)  
+* [Broker] Allow to enable/disable delayed delivery for messages on namespace [#5915](https://github.com/apache/pulsar/pull/5915)  
+* [Broker] Prevent creation of regular topic with the same name as existing partitioned topic [#5943](https://github.com/apache/pulsar/pull/5943)  
+* [Broker] Reset cursor with a non-exists position [#6120](https://github.com/apache/pulsar/pull/6120)  
+* [Broker] Use fully qualified hostname as default to advertise brokers [#6235](https://github.com/apache/pulsar/pull/6235)  
+* [broker] Timeout API calls in BrokerService [#6489](https://github.com/apache/pulsar/pull/6489)  
+* [Broker] Fix some async method problems at PersistentTopicsBase. [#6483](https://github.com/apache/pulsar/pull/6483) 
+* [Broker] Start namespace service and schema registry service before start broker. [#6499](https://github.com/apache/pulsar/pull/6499)  
+* [Broker] Disable channel auto read when publish rate or publish buffer exceeded [#6550](https://github.com/apache/pulsar/pull/6550)  
+* [BRoker] Resume some servercnx method to public [#6581](https://github.com/apache/pulsar/pull/6581)  
+
+
+
+#### Pulsar Proxy
+* [Proxy] Fixed correct name for proxy thread executor name [#6460](https://github.com/apache/pulsar/pull/6460)  
+* [Proxy] Fixed logging for published messages [#6474](https://github.com/apache/pulsar/pull/6474)  
+* [Proxy] Fixed proxy routing to functions worker [#6486](https://github.com/apache/pulsar/pull/6486)  
+
+
+#### Stats & Monitoring 
+* [Broker] Add backlogSize in topicStats [#5914](https://github.com/apache/pulsar/pull/5914)  
+* [Broker] Expose lastConsumedTimestamp and lastAckedTimestamp to consumer stats [#6051](https://github.com/apache/pulsar/pull/6051)  
+* Improve backlogSize stats in the topic. [#6700](https://github.com/apache/pulsar/pull/6700)  
+
+#### Security
+* Fixed publish buffer limit does not take effect [#6431](https://github.com/apache/pulsar/pull/6431)  
+* Fixed the bug of authenticationData is't initialized. [#6440](https://github.com/apache/pulsar/pull/6440)  
+* [Authentication] Validate tokens for binary connections [#6233](https://github.com/apache/pulsar/pull/6233)  
+* [Broker] Create namespace failed when TLS is enabled in PulsarStandalone [#6457](https://github.com/apache/pulsar/pull/6457) 
+* Use more granular permissions for topics [#6504](https://github.com/apache/pulsar/pull/6504)  
+
+
+#### Pulsar Schema
+
+* [Schema] Fixed get schema version in HttpLookupService. [#6193](https://github.com/apache/pulsar/pull/6193)  
+* [Schema] Independent schema is set for each consumer generated by topic [#6356](https://github.com/apache/pulsar/pull/6356)  
+* [Schema] Fixed wrong avro schema [#6406](https://github.com/apache/pulsar/pull/6406)  
```

Comment:
Need a meaningful name for this fix. You can pick some key sentences in the description of this PR. Sometimes the title of the PR did not express key messages.